packages/sdk-core/src/modules/breadcrumbs/BreadcrumbsManager.ts

```diff
@@ -148,7 +178,32 @@ export class BreadcrumbsManager implements BacktraceBreadcrumbs, BacktraceModule
             return false;
         }
 
-        this._storage.add(rawBreadcrumb);
+        if (this._limits.maximumBreadcrumbMessageLength !== undefined) {
+            rawBreadcrumb = {
+                ...rawBreadcrumb,
+                message: rawBreadcrumb.message.substring(0, this._limits.maximumBreadcrumbMessageLength),
+            };
```

Comment:
small nit:
```suggestion
            rawBreadcrumb.message = rawBreadcrumb.message.substring(0, this._limits.maximumBreadcrumbMessageLength);
```
I think it should be enough, right?

Reply:
yes, changed